firebase-installations/src/main/java/com/google/firebase/installations/Utils.java

```diff
@@ -43,7 +48,12 @@ public boolean isAuthTokenExpired(PersistedInstallationEntry entry) {
 
   /** Returns current time in seconds. */
   public long currentTimeInSecs() {
```

Comment:
static?

Reply:
https://github.com/firebase/firebase-android-sdk/pull/1954/files#r487182669

Reply:
Please add a respective comment (not JavaDoc) to the method.